src/components/admin/Api.tsx

```diff
@@ -43,7 +43,13 @@ function AdminApi() {
                 <TextareaAutosize
                     minRows={4}
                     cols={50}
-                    value={session?.access_token}
+                    value={JSON.stringify({
+                        access_token: session?.access_token,
+                        refresh_token: session?.refresh_token,
+                        expires_at:
+                            (session?.expires_at ?? 0) +
+                            (session?.expires_in ?? 0),
+                    })}
```

Comment:
I think we can just generally check for session as the start

```
value={session ? JSON.stringify({
    access_token: session.access_token,
    refresh_token: session.refresh_token,
    expires_at:
        (session.expires_at ?? 0) +
        (session.expires_in ?? 0),
}) : ""}
```

The session "can be null" but that usually just means they get logged out right away. If we are missing session then we can just not show anything in the text area.